package.json

```diff
@@ -8,7 +8,8 @@
     "prepublishOnly": "npm run build",
```

Comment:
npm script `prepublishOnly` has been removed.  `npm run build` before  publish package seems curt,  `npm run test` should be better if there must be a `prepublishOnly` script.